utils/config-utils.js

```diff
@@ -16,9 +16,12 @@ export async function initCuttleGlobals(app) {
   // Connect the devtools -- non-prod only
   if (import.meta.env.DEV) {
     try {
-      const devtools = await import('@vue/devtools');
-      devtools.connect(null, 8098);
-      console.log('Vue devtools connected');
+      document.addEventListener('DOMContentLoaded', () => {
+        const script = document.createElement('script');
+        script.src = 'http://localhost:8098';
+        script.async = true;
+        document.head.appendChild(script);
+      });
```

Comment:
This works with the new version of devtools better than the previous method.

Reply:
Is it catchable? Historically we haven't been actually able to hit that `console.warn()` when the devtools fail to connect. It would be nice if we could swallow whatever vue would output and just send this warning ourselves. Either way it wouldn't be a regression b/c the catch never worked